lib/mayaUsd/fileio/jobs/jobArgs.h

```diff
@@ -282,6 +284,10 @@ struct UsdMayaJobImportArgs
     /// non-empty interval.
     const GfInterval timeInterval;
 
+    typedef std::map<std::string, std::string> ChaserArgs;
```

Comment:
`using` vs. `typedef`

Reply:
This is addressed in e8efa0c.

Reply:
You changed export, but not import.

Reply:
Apologies, you're correct. This is now addressed in ef5450c.